iterations_runners/iterations_runner.c

```diff
@@ -15,11 +15,14 @@
 #include <dlfcn.h>
 #include <err.h>
 #include <string.h>
+#include <inttypes.h>
 
 #define BENCH_FUNC_NAME "run_iter"
+#define TSR_DEFAULT 0xffffffffffffffff
```

Comment:
When you do 0xffffffffffffffffff, it makes people think it means something important (at first, I assumed it was a bit mask). So let's make it something that's "obviously" neutral.